### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Iragemini**
work on bugs

### CODE REVIEW

File: src/breakout/Board.tsx [hunk 1]
```diff
@@ -24,13 +24,13 @@ const Board: React.FC = () => {
 
   useEffect(() => {
     const render = () => {
-      const canvas: HTMLCanvasElement = canvasRef.current;
-      const ctx: CanvasRenderingContext2D = canvas.getContext('2d');
+      const canvas = canvasRef.current as HTMLCanvasElement;
```
Comment (taraswww777) [inline src/breakout/Board.tsx:27]: Зачем?

File: src/breakout/Brick.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+import Colors from '../Colors';
+
 type BrickParams = {
   x: number;
   y: number;
   width: number;
   height: number;
-  colors: any;
-};
+  colors: string[];
```
Comment (taraswww777) [inline src/breakout/Brick.ts:9]: colors: Colors[]

File: src/breakout/Brick.ts [hunk 3]
```diff
@@ -24,26 +27,26 @@ const brick = (level: number, bricks: [], canvas: HTMLCanvasElement, brick: Bric
...
 };
 
 class SingleBrick {
   private x: number;
   private y: number;
   private width: number;
   private height: number;
-  private colors: String[];
-  private broke: boolean;
+  private colors: string[];
```
Comment (taraswww777) [inline src/breakout/Brick.ts:46]: colors: Colors[]

File: src/breakout/Board.tsx (hunk 1 above)
Comment (Iragemini) [reply src/breakout/Board.tsx:27]: +

File: src/breakout/Brick.ts (hunk 2 above)
Comment (Iragemini) [reply src/breakout/Brick.ts]: done

File: src/breakout/Brick.ts (hunk 3 above)
Comment (Iragemini) [reply src/breakout/Brick.ts]: done

**Pull request merged**